prov/kubernetes/execution.go

```diff
@@ -275,14 +277,15 @@ func (e *executionCommon) manageDeploymentResource(ctx context.Context, clientse
 
 		events.WithContextOptionalFields(ctx).NewLogEntry(events.LogLevelINFO, e.deploymentID).Registerf("k8s Deployment %s deleted", deploymentName)
 
-		// Delete namespace
-		err = generator.deleteNamespace(namespace, clientset)
-		if err != nil {
-			events.WithContextOptionalFields(ctx).NewLogEntry(events.LogLevelINFO, e.deploymentID).Registerf("Cannot delete %s k8s Namespace", namespace)
-			return err
+		// Delete namespace if not provided
```

Comment:
Maybe not related to this US but what if we have several deployments in the same app?
This is generally the case. Like in the monitoring sample. We will wait for the deletion of graphite then try to delete the namespace while grafana's deployment is still there.

Reply:
Maybe the same namespace should be used for all the related deployments (graphite and grafana in the monitoring sample).

Reply:
I mean this problem does not arrive if we use the current new feature that allows to specify the namespace for all the k8s objects of the same app